frontend/src/pages/Registration.jsx

```
@@ -0,0 +1,270 @@
+
+  // Watch the password field to detect changes
+  const password = watch('password');
+
+  return (
+    <section className="min-h-screen bg-green-100 flex items-center justify-center p-4">
+      <div className="bg-blue-100 p-8 rounded-lg shadow-lg w-full max-w-md">
+        <h1 className="text-2xl font-bold mb-6 text-center">Register</h1>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          {/* Full Name */}
+          <div className="mb-4">
+            <label className="block text-sm font-medium mb-2">Full Name</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)